walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -22,40 +22,42 @@
 
 public final class FindMeetingQuery {
 
-  public final int duration = 0;
-  public final Boolean notInclusive = false;
-  public final Boolean inclusive = true;
+  public final int DURATION = 0;
```

Comment:
Nit: A more descriptive name would be useful here, like EMPTY_DURATION or DEFAULT_DURATION